robottelo/rh_cloud_utils.py

```diff
@@ -100,3 +102,10 @@ def get_report_data(report_path):
             if file_name != 'metadata.json':
                 json_data = json.load(tarobj.extractfile(file_))
     return json_data
+
+
+def setting_update(name, value):
```

Comment:
I don't think this is a great pattern to have this 'helper' function here. This to me looks like a core `Capsule` instance method, that could reference `self.api.Setting()`

Reply:
@mshriver I didn't get it. Can you tell me what exactly I need to change?

Reply:
So, you have a function that's using `entities`, which is assuming the nailgun ServerConfig based on reading settings.server.hostname - implicit behavior that we're trying to remove from robottelo.

Updating a setting is something that is likely to be a common action in the test session, and this entity is used about a dozen times across the repo.

Instead of creating this helper method with very small scope and functionality where no one else is likely to find it, for use within your test module for rh_cloud, invites this function to be re-defined multiple places as people want to make settings updates.

Instead, you could make this a method on the `robottelo.hosts.Satellite` class, so that it is visible, discoverable, and re-usable across the framework.  Its implementation will also be more explicit, as it should be able to reference `self.api.Setting()` to get the settings instance, instead of using the implicit ServerConfig when entities classes are imported and referenced.

Then within tests, anyone can reference `default_sat.update_setting(name=<key>, value=<something>)` without having to import anything, or make implicit assumptions about what satellite is getting its settings updated.